portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Icon.path

```diff
@@ -414,6 +414,11 @@
 	<td>//tr[contains(.,'${key_rowEntry}')]//a[span/*[name()='svg'][contains(@class,'icon-ellipsis')]]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SVG_ELLIPSIS</td>
+	<td>//*[name()='svg'][contains(@class,'icon-ellipsis')]</td>
```

Comment:
I think this is a smell that something's not right... if I'm counting correctly, this is the 5th `*_ELLIPSIS` path, counting both vertical and regular ellipsis. 

The smell is that we're targetting implementation details and not a functional representation. All these paths somehow attempt to open an "options" menu. I think this is one obvious place where being more conscious about tests could improve the semantics and consistency of our whole platform.

Reply:
Yep...I've tried to find a suitable existing ELLIPSIS path to target the one in Remote Apps but I couldn't find a pattern that worked for this case.